docs/commands/hexdump.md

```diff
@@ -2,10 +2,11 @@
 
 Imitation of the WinDBG command.
 
-This command requires at least 2 arguments, the format for representing the
-data, and a value/address/symbol used as the location to print the hexdump
-from. An optional 3rd argument is used to specify the number of
-qword/dword/word/bytes to display.
+This command takes 4 optional arguments, the format for representing the
```

Comment:
This would be more readable if shown as a list because there are many arguments.

Reply:
I agree. I have updated it as a list with the recent commit.